feature-styles/HY_P_Watercourse_ManMade.se

```diff
@@ -107,7 +107,7 @@ version="1.1.0">
     </se:Rule>
     <se:Rule>
         <se:Description>
-      <se:Title>waterbodies man-made</se:Title>
+      <se:Title>Waterbodies man-made</se:Title>
```

Comment:
For other cases with different geometries, the geometry was mentioned in the title - maybe align?